MonoGame.Framework/Matrix.cs

```diff
@@ -984,12 +984,12 @@ public static void CreatePerspectiveFieldOfView(float fieldOfView, float aspectR
 		    if (nearPlaneDistance >= farPlaneDistance)
 		    {
 		        throw new ArgumentException("nearPlaneDistance >= farPlaneDistance");
-		    }
-
-            var yScale = 1.0f / (float)Math.Tan((double)fieldOfView * 0.5f);

```

Comment:
Precision lost!
Here '0.5f' was upgraded to double, because fieldOfView was explicitly converted to double, and the multiplication was done with double precision.
Then Tan() was calculated with double precision.